about.html

```diff
@@ -0,0 +1,132 @@
+            <div class="about-program-header">
+                <h3 class="about-text">Creative Commons Global Summit 2015 Logo</h3>
+                <hr>
+            </div>
+            <div>
+                <p class="about-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed molestie ante, a semper magna. Nam quis iaculis libero. Sed vehicula, augue et dapibus ornare, magna risus aliquet ipsum, in imperdiet ex arcu ac risus. Cras porta aliquam
+                    porttitor. Nunc sed tortor ipsum. Cras suscipit ex risus, sed consequat quam aliquet id. Cras hendrerit suscipit odio, sit amet venenatis eros vestibulum consectetur. Integer at enim sagittis, lacinia massa nec, condimentum ex.</p>
+            </div>
+            <div class="about-image">
+                <img src="./images/creative-commons-license-logo.jpeg" alt="logo">
+            </div>
+            <hr>
+        </section>
+
+        <section>
+            <div class="past-summit">
+                <h3 class="past-summit-header">See the past CC Global Summits</h3>
+                <hr>
+                <p class="past-summit-text">Take a look at the last two summits which took place in buenos aires</p>
+            </div>
+
+            <div class="summit">
+                <div class="summit-1">
+                    <div class="overlay-red">
+                        <h3>2013</h3>
+                        <p>CC Global summit buenos aires</p>
+                    </div>
+                </div>
+                <div class="summit-2">
+                    <div class="overlay-red">
+                        <h3>2013</h3>
+                        <p>CC Global summit buenos aires</p>
+                    </div>
+                </div>
+            </div>
+        
+        <section>
+            <section class="partners">
+                <div class="partner-header">
+                    <h3>Partners</h3>
+                    <hr>
+                </div>
+                <div class="partner-logo">
+                    <a class="bold">mozilla</a>
+                    <a class="faint">Google</a>
+                    <a class="bold">NAVER</a>
+                    <a class="faint">daumkakao</a>
+                    <a class="bold"><i class="fa fa-airbnb"></i>airbnb</a>
+                </div>
+            </section>
+
+        </section>
+
+       
+        </section>
+    </main>
+
+    <footer class="footer">
+        <div class="footer-content">
+            <img src="images/creative-commons-license-logo.jpeg" class="footer-img" alt="logo">
+            <div class="footer-text">
+                <p>Copyright © 2015 Creative Commons Korea. Some rights reserved.</p>
+                <p class="licence-text">This work is licensed under a <a class="link" rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a></p>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] The footer in the desktop version should have dark background please try to do that